discovery-provider/src/__init__.py

```diff
@@ -288,7 +288,7 @@ def configure_celery(flask_app, celery, test_config=None):
                  "src.tasks.index_plays", "src.tasks.index_metrics",
                  "src.tasks.index_materialized_views",
                  "src.tasks.index_network_peers", "src.tasks.index_trending",
-                 "src.tasks.cache_user_balance"
+                 "src.tasks.cache_user_balance", "src.monitors.monitoring_queue"
```

Comment:
moving forward, let's be more thoughtful about adding tasks here and including folks in the design decision to do so - i don't think this is a problem yet, but over time i suspect there may be degradation of some kind if we just keep adding tasks instead of consolidating logic where possible